integration-cli/docker_api_stats_test.go

```diff
@@ -69,3 +71,41 @@ func (s *DockerSuite) TestStoppedContainerStatsGoroutines(c *check.C) {
 		}
 	}
 }
+
+func (s *DockerSuite) TestApiNetworkStats(c *check.C) {
+	// Run container for 30 secs
+	out, _ := dockerCmd(c, "run", "-d", "busybox", "/bin/sh", "-c", "sleep 30")
+	id := strings.TrimSpace(out)
+	err := waitRun(id)
+	c.Assert(err, check.IsNil)
+
+	// Retrieve the container address
+	contIP := findContainerIP(c, id)
+	numPings := 10
+
+	// Get the container networking stats before and after pinging the container
+	nwStatsPre := getNetworkStats(c, id)
+	_, err = exec.Command("ping", contIP, "-c", strconv.Itoa(numPings)).Output()
```

Comment:
The purpose of this test is to trigger a packet exchange with the container from outside of the container, in a minimal way so that we know which packets are being exchanged.

If we spawn another container, then the previous container would receive broadcast advertisement frames and arps from the new container. Which is all noise for what this test is trying to assert.

Therefore I'd prefer to keep this logic.